### PR TITLE
Add documentation about aXe

### PR DESCRIPTION
**Pull request by jkassemi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,21 +223,34 @@ the test requries features that jsdom does not provide, putting them into a
...
 
 ### Continuous Integration
 Continuous integration and deployment is done via
 [Travis CI](travis-ci.org/department-of-veterans-affairs/vets-website). All of the configuration
 is stored in `.travis.yml`.
 
-All pushes kick off a matrix build for both `BUILDTYPE=development` and `BUILDTYPE=production`.
-Ensures that all PRs always work on both build configurations.
+The build configuration will depend on the branch being pushed. The `master`
+brnach and any feature branches will trigger a build with the default build type
```
Comment (crwallace) [inline README.md]: typo `brnach`.

Comment (jkassemi) [inline README.md]: nice catch, I need to look into a spell checker 😄

**Comment by crwallace** [comment]
@jkassemi -- Is this ready for merge and review?

**Comment by jkassemi** [comment]
Yep, don't hesitate to let me know if you'd like more detail

**Pull request merged**